R/check_suggested.R

```diff
@@ -10,27 +10,25 @@
 #' @noRd
 check_suggested <- function(packages,
                             use,
-                            top_level_fun = NULL,
-                            message = NULL,
-                            version = NULL) {
+                            top_level_fun = NULL
+                            ) {
   # If NA, inform that package isn't installed, but only in interactive mode
   if (is.na(use)) {
     use <- is_interactive()
     if (!use) {
       return(FALSE)
     }
 
-    installed <- map_lgl(packages, ~ {
-      installed <- is_installed(., version = version)
-      if (!installed) {
-        if (is.null(message)) {
-          # FIXME: Mention version
-          message <- glue("`{top_level_fun}()` is improved by the '{.}' package. Consider `install.packages(\"{.}\")`.")
-        }
-        inform(message)
-      }
-      installed
+    installed <- map_lgl(packages, function(pkg) {
+      is_installed(pkg)
     })
+
+    if (any(!installed)) {
+      pkgs_not_installed <- packages[!installed]
+      message <- "{.fn {top_level_fun}} is improved by the '{.pkg {pkgs_not_installed}}' package. Consider installing them."
```

Comment:
How can we distinguish between singular and plural here? What happens if more than one package is not installed?

Reply:
Done. https://cli.r-lib.org/reference/pluralization.html